pkg/auth/service.go

```diff
@@ -1439,9 +1451,13 @@ func (a *APIAuthService) ListGroupUsers(ctx context.Context, groupDisplayName st
 	members := make([]*model.User, len(resp.JSON200.Results))
 
 	for i, r := range resp.JSON200.Results {
+		username := r.Username
+		if username == "" {
+			username = swag.StringValue(r.Name)
```

Comment:
We would like to return the Name as Username even if it is empty, as we will make it required